README.md

```diff
@@ -10,6 +10,8 @@
 
 Try Medium with membership-features!
 
+**If you are using Firefox, must allow this extension in incognito mode!**
```

Comment:
HI Johnson, can you shed more light on this?

Reply:
The line below requires the extension to be allowed in incognito mode. (Firefox have a option now to specify whether the extension is allowed in incognito mode)
```
browser.windows.create({"url": url, "incognito": true});
```

Reply:
After installing the extension, set `Tools > Add-ons > Mediumship > Run in Private Windows` to `Allow`

![image](https://user-images.githubusercontent.com/20457146/59119541-a990b700-8985-11e9-8863-e8e597d09ab8.png)

Reply:
Sorry I didn't understand. 😅
In the normal mode, clicking on this extension opens the tab in incognito mode. Why does the extension be allowed in incognito mode?

Reply:
Oh, I think I didn't express it clear enough.
Seems like `opening a window in incognito mode` requires the extension to be allowed in incognito mode in firefox.

Reply:
Got it! This behavior is weird though.

Reply:
I got quite confused at first too. I found the error message in the extension debugging console and solved it by allowing the extension in incognito mode. I'm wondering if there are other ways to open an incognito tab without the permission.

Reply:
So I got to rewrite the whole thing from scratch, and we have v2.0 now! It's being published in Chrome Webstore right now. Once we verify that everything works as expected, maybe we can do the same for the Firefox version.
P.S. The new version works wholly in background, without any user-intervention! 😉

Reply:
2.0 looks good to me. Can you check it out at your end? Let's bump up the firefox version too! 👊

Reply:
We can post to Product Hunt at midnight (13 hours from now): https://whattimeisitatproducthunt.com/

Reply:
Checking in if you can make it within the next 4 hours... 😅

Reply:
I've just finished reading the code. I think maybe we should keep the user-clicking action on any other domain name?

Reply:
By that way we can fix #6, if the domain name isn't medium.com

Reply:
Yes, makes sense! 👍
https://github.com/swapagarwal/mediumship/issues/6#issuecomment-500234833

Reply:
OK. I'll try to update it in the next 2 hours.

Reply:
Awesome! Send me your PH username if you'd like to be added as a maker [here](https://www.producthunt.com/posts/mediumship).

Reply:
My PH username is `j3soon`. By the way, what's this url `https://t.co/JV5396gd2O` pointing to?

Reply:
It points to [medium.com](http://www.getlinkinfo.com/info?link=https://t.co/JV5396gd2O) 😏
(Added on [PH](https://www.producthunt.com/posts/mediumship))

Reply:
But I think we're using twitter as the referrer as mentioned in #5 ?

Reply:
So that means using any Twitter URL as the referrer, e.g. `https://t.co/JV5396gd2O` and not Twitter itself. 😅

Reply:
Got it! I'll close this PR and open another one.